.env

```diff
@@ -18,8 +18,11 @@ JWT_SECRET=1c053cacaa6aa0963e211cdc7cb5b5f4bc0d99cb9b7b5aca0c1f16f104102486053e8
 
 STRIPE_PUBLISHABLE_KEY=pk_test_aQbfVWeOwvtES5FRSY7iIjk9
 STRIPE_SECRET_KEY=sk_test_Qk8TgJVXgeC1xJWu0tmwCW0l
-STREAM_API_KEY=sjm3sx9mgcx2
-STREAM_API_SECRET=uxupxkuw5ckrd2fe9qz88zne4qagwmdm9yaegbsm8xuymqja6ff8jr678636v3c4
+STREAM_API_KEY=k3cdhuxr9576
+STREAM_API_SECRET=bde4gr556yx5rrmsknwn3vfaph5juybkrp9xwhwrtvth2n8ugjuwh5wapatbs9vp
+
+ONE_SIGNAL_APP_ID=8ae45028-2f4a-4ead-8b5f-cc9dfaa74058
+ONE_SIGNAL_API_KEY=MjhhMGQ0YTgtN2ZhZC00MzY5LWE1ODEtN2M2MDk2M2Q5MDcx
```

Comment:
Yeah, shouldn't put sensitive information in the .env file. 
Those are development keys and tests can run without them.